modules/reference/pages/tunable-properties.adoc

```diff
@@ -1372,6 +1372,38 @@ Maximum size of the user-space receive buffer. If `null`, this limit is not appl
 
 ---
 
+[[kafka_throughput_throttling_v2]]
+=== kafka_throughput_throttling_v2
+
+This disables version two of throughput throttling, introduced with Redpanda v23.3.8. When this is disabled, Redpanda uses the throttling mechanism from v23 and older. This defaults to true for all new or upgraded Redpanda clusters.
+
+*Type*: boolean
+
+*Default*: true
+
+*Restart required*: true
+
+WARNING: Disabling this property causes your Redpanda cluster to use an outdated throughput throttling mechanism. Only set this to `false` when advised to do so by Redpanda support.
+
+---
+
+[[kafka_throughput_replenish_threshold]]
+=== kafka_throughput_replenish_threshold
+
+Threshold for refilling the token bucket. Will be clamped between 1 and rate.
+
+*Type*: unsigned integer
```

Comment:
It's a signed 64bit integer.
https://github.com/redpanda-data/redpanda/blob/v23.3.8/src/v/config/configuration.h#L523C36-L524